### PR TITLE
Footersocial styles #6007

### PR DESCRIPTION
**Pull request by DipakShrestha18**

### CODE REVIEW

File: src/components/Footer/FooterSocial/FooterSocialItem.tsx [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
-    <a href={url} rel="noopener noreferrer" target="_blank">
+    <a
+      href={url}
+      className="footer-social-item"
+      rel="noopener noreferrer"
+      target="_blank"
+    >
+      <Icon
+        className="footer-social-item__icon"
+        name={text.charAt(0).toLowerCase() + text.slice(1)}
```
Comment (creido-welly) [inline src/components/Footer/FooterSocial/FooterSocialItem.tsx:20]: could simplify to `text.toLowerCase()`

File: src/components/Footer/styles/_35-footer-links.scss [hunk 2]
```diff
@@ -21,6 +21,7 @@
   margin: 0 var(--space-unit);
   position: relative;
   text-decoration: none;
+  @include animated-underline;
```
Comment (creido-welly) [inline src/components/Footer/styles/_35-footer-links.scss:24]: put the `@include` at the top of the style block, after any `@extend`s

File: src/components/Footer/FooterSocial/FooterSocialItem.tsx (hunk 1 above)
Comment (DipakShrestha18) [reply src/components/Footer/FooterSocial/FooterSocialItem.tsx:20]: I just need to make first character lowercase. There is an item with the uppercase in the middle of string which should not be transformed.

File: src/components/Footer/styles/_35-footer-links.scss (hunk 2 above)
Comment (DipakShrestha18) [reply src/components/Footer/styles/_35-footer-links.scss:24]: done. re-review pls.

Comment (chriddell) [reply src/components/Footer/styles/_35-footer-links.scss:24]: This should be caught by the style-lint, I'll investigate

Comment (DipakShrestha18) [reply src/components/Footer/styles/_35-footer-links.scss:24]: I did do a lint checking, there was no error.

Comment (chriddell) [reply src/components/Footer/styles/_35-footer-links.scss:24]: yep I know - but there _should_ be an error 😬 writing some new lint rules now https://github.com/hudochenkov/stylelint-order/blob/master/rules/order/README.md

File: src/components/Footer/FooterSocial/FooterSocialItem.tsx (hunk 1 above)
Comment (creido-welly) [reply src/components/Footer/FooterSocial/FooterSocialItem.tsx:20]: 👍

File: src/components/Footer/styles/_35-footer-links.scss (hunk 2 above)
Comment (chriddell) [reply src/components/Footer/styles/_35-footer-links.scss:24]: Any thoughts on https://github.com/wellcometrust/corporate/issues/6035 welcomed

**Review by creido-welly — APPROVED** [review]

**Pull request merged**